docs/openid.md

```diff
@@ -52,6 +52,31 @@ specific providers.
 
 Here are a few configs for providers that should work with Synapse.
 
+### Microsoft Azure Active Directory
+Azure AD can act as an OpenID Connect Provider. Register a new application under 
+*App registrations* in the Azure AD management console. The RedirectURI for your
+application should be set to your matrix server: `https://matrix.example.com/_synapse/oidc/callback`
+
+Go to Certificates & secrets and register a new client secret. Make not of your Directory (tenant) ID.
+Edit your /etc/matrix-synapse/homeserver.yaml file and change the oidc_config section:
```

Comment:
I believe the other examples just say to edit your Synapse config. (Since it might not exist in that location.)

Reply:
Thx! Fixed i updated  version.